client/client-multi/client-c/src/platform/system_logger.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * system_logger.h
+ *
+ *  Created on: Dec 31, 2014
+ *      Author: avp
+ */
+
+#ifndef SRC_KAA_UTILITIES_SYSTEM_LOGGER_H_
+#define SRC_KAA_UTILITIES_SYSTEM_LOGGER_H_
+
+#include "../kaa_error.h"
+
+void write_log(FILE * sink, const char * buffer, size_t message_size);
```

Comment:
Add doxygen documentation to every function. See platform/ext_log_storage.h for an example